src/Normalizer/ImageFileNormalizer.php

```diff
@@ -65,6 +65,8 @@ public function normalize($entity, $format = NULL, array $context = []) {
       'image_with_caption',
       'image_wide_with_caption',
       'image_thumbnail_with_caption',
+      'anu_lms_paragraph_image_bullet_list_small',
```

Comment:
We need to keep existing image styles naming convention, I'd recommend to rename it to `image_bullet_list_small` or `image_bullet_list_small` (same for name below). 
You can rename it everywhere (in code and file names) and import config (make drush cim).